scripts/dkim_registry.ts

```diff
@@ -0,0 +1,134 @@
+// This script calls setDKIMPublicKeyHash or revokeDKIMPublicKeyHash in UserOverridableDKIMRegistry on zksync sepolia
+// Usage: npx ts-node scripts/dkim_registry.ts --command set --private-key <sender-private-key> --authorizer-private-key <authorizer-private-key>
+
+import { Command } from 'commander';
+import { Contract, ethers } from 'ethers';
+
+const ZKSYNC_SEPOLIA_RPC_URL = 'https://sepolia.era.zksync.dev';
+const ZKSYNC_SEPOLIA_CHAIN_ID = 300;
+
+// TODO: Please modify the following values according to your specific use case
+const USEROVERRIDEABLE_DKIM_REGISTRY_ADDRESS = "0x037d80f98aE461DC307F16524EC0Fc53b2E9E0b1"; // For zksync sepolia
+const DOMAIN_NAME = "google.com";
+const PUBLIC_KEY_HASH = "0x0ea9c777dc7110e5a9e89b13f0cfc540e3845ba120b2b6dc24024d61488d4784";
```

Comment:
@wshino Could you pass `USEROVERRIDEABLE_DKIM_REGISTRY_ADDRESS`, `DOMAIN_NAME`, and  `PUBLIC_KEY_HASH ` as arguments to the script, rather than fixed values?
Please write the "gmail.com" and its correct public key hash as example values of the arguments in the comment.

Reply:
@SoraSuegami Updated